lib/account/account-manager.js

```diff
@@ -110,10 +313,10 @@ module.exports = function (accountService, config, hotp, otDate, promise, random
 
         accountInfo = {};
         accountInfo.accountId = accountId;
-        accountInfo.password = registrationInfo.password;
+        accountInfo.password = base64.decode(registrationInfo.password);
```

Comment:
In your directions at https://ide.c9.io/fidian/api-examples in the login.md on line 39 it says: "Note that we decode the base64 and save the raw buffer with the account." So I have to decode the password/hash and save that in the account file.